apps/examples/stack_protection/Kconfig

```diff
@@ -0,0 +1,14 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_STACK_PROTECTION
+	bool "Stack Protection TestCase Example"
+	depends on MPU_STACK_OVERFLOW_PROTECTION
+	---help---
+		Enable the Stack Overflow Protection TestCase Example
+
+config USER_ENTRYPOINT
+	string
+	default "stack_protection_main" if ENTRY_STACK_PROTECTION
```

Comment:
Default entry point name not in line with C file function name `stack_prot_tc_main`

Reply:
changed